src/features/page_view_event/aggregate/index.js

```diff
@@ -69,6 +69,8 @@ export class Aggregate extends AggregateBase {
       at: info.atts
     }
 
+    if (agentRuntime.session) queryParameters.fsh = Number(agentRuntime.session.isNew) // "first session harvest" aka RUM request or PageView event of a session
```

Comment:
Everything looks good, just one quick philosophical question here,  Do we want to always send the `fsh` (like shown here), or should we just only send it if its truthy?

Reply:
If it's not sent at all, there's no session tracking running; distinct from sending '0' ig. That might matter if you're trying to find statistics.

Also does downstream need to be aware of this going as `fsh` before merge? Does it matter what it is as long as they know the exact string?

Reply:
They have not done the work. It will just have no effect (dropped) until they implement the property on their end